Gopkg.toml

```diff
@@ -0,0 +1,23 @@
+[[constraint]]
+  branch = "master"
```

Comment:
change to v0.1, see https://github.com/kubernetes-csi/drivers/pull/36/files#diff-836546cc53507f6b2d581088903b1785R26

Reply:
done